### PR TITLE
Remove Trustyai installation in downstream

### PR DESCRIPTION
**Pull request by VaishnaviHire**
Jira Issue: https://issues.redhat.com/browse/RHOAIENG-3391

Test Catalogssource: quay.io/vhire/opendatahub-operator-catalog:v2.7.0

### CODE REVIEW

**Review by ruivieira — APPROVED** [review]
LGTM.

@VaishnaviHire I'm assuming removing the component, but keeping the alerting rules won't be an issue?

**Comment by zdtsw** [comment]
> quay.io/vhire/opendatahub-operator-catalog:v2.7.0



> LGTM.
> 
> @VaishnaviHire I'm assuming removing the component, but keeping the alerting rules won't be an issue?

it is set to disabled, so the alerts wont be added into SRE prometheus regardless what value users set in DSC

File: controllers/status/status.go [hunk 1]
```diff
@@ -190,6 +197,11 @@ func SetComponentCondition(conditions *[]conditionsv1.Condition, component strin
 
 // RemoveComponentCondition remove Condition of giving component.
 func RemoveComponentCondition(conditions *[]conditionsv1.Condition, component string) {
-	condType := component + ReadySuffix
+	var condType string
+	if component == trustyai.ComponentName {
+		condType = component + "Deprecated"
+	} else {
```
Comment (zdtsw) [inline controllers/status/status.go:203]: think we should not set this, 
eventually this status will be removed so not leave a TrustyAIDeprecated in the condition.

Comment (VaishnaviHire) [reply controllers/status/status.go:203]: Okay I will update this

**Review by zdtsw — APPROVED** [review]

**Pull request merged**